src/test/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionaryIntegrationTest.java

```diff
@@ -35,6 +35,22 @@ public void testBasic() throws Exception {
         IntegrationTestSpec.assertEqualTextFiles(outputDict, new File(BASIC_FASTA.getAbsolutePath() + ".dict"));
     }
 
+    @Test
+    public void testNoOutput() throws Exception {
```

Comment:
Can you change the name of this method to something that reflects the fact that its using default output ? "testNoOutput" sounds like it does something completly different..